artistools/initial_composition.py

```diff
@@ -56,14 +56,25 @@ def plot_2d_initial_abundances(modelpath, args):
 
 def get_merged_model_abundances(modelpath):
     # t_model is in days and vmax is in cm/s
-    model, t_model, vmax = at.inputmodel.get_modeldata_tuple(modelpath[0], dimensions=3)
+    model, t_model_days, vmax = at.inputmodel.get_modeldata_tuple(modelpath[0], dimensions=3)
+
+    targetmodeltime_days = None
+    if targetmodeltime_days is not None:
```

Comment:
Something is missing here

Reply:
To be added at some point. To make my plot I've just changed it from None.

Reply:
To be added at some point. To make my plot I've just changed it from None.